dist/index.js

```diff
@@ -3337,20 +3337,10 @@ function run() {
             if (!validateDistros(targetRos1Distro, targetRos2Distro)) {
                 return;
             }
-            // Source any installed ROS binary distribution, safely creating an empty setup file if it is not present
-            let commandPrefix = "";
-            if (isLinux) {
-                if (targetRos1Distro) {
-                    commandPrefix += `mkdir -p /opt/ros/${targetRos1Distro} && touch /opt/ros/${targetRos1Distro}/setup.sh && source /opt/ros/${targetRos1Distro}/setup.sh && `;
-                }
-                if (targetRos2Distro) {
-                    commandPrefix += `mkdir -p /opt/ros/${targetRos2Distro} && touch /opt/ros/${targetRos2Distro}/setup.sh && source /opt/ros/${targetRos2Distro}/setup.sh && `;
-                }
-            }
-            // rosdep on Windows does not reliably work on Windows, see
+            // rosdep does not reliably work on Windows, see
```

Comment:
Is this comment incomplete?

Reply:
Isn't this continued on the next line?

Reply:
yeah i was just removing the redundant "on Windows" from the sentence, it continues on the next line